src/shaders/_prelude_fog.fragment.glsl

```diff
@@ -35,7 +36,11 @@ float fog_opacity(vec3 position) {
 }
 
 vec3 fog_apply(vec3 color, vec3 position) {
```

Comment:
Should we have two variants of this, e.g. fog_apply_dithered and fog_apply? Just to switch to a different one for anything text/icon related. Making it explicit in the name makes it easier to not use the dithered one by accident.

Reply:
Would we only apply dithering to shaders with features which take up large portions of the screen? (i.e. exclude circles, lines, symbols……?)

Reply:
And anyway, in this case, it's probably best not to include dithering in fog at all and instead just call it manually where desired.

Reply:
The ones that seem to naturally fit the _non-dithered_ version seems to be the ones that have a single fog falloff for their entire drawn geometry, whether it's point based or a quad facing the camera. I can only think of circles, symbols and maybe heatmap. Lines may still have large geometry based on the line width and high pitch.

Reply:
I've moved this into a separate function for now so that it more clearly remains a choice of each shader if and how it requires dithering.